### PR TITLE
Added toml plugin

### PR DESCRIPTION
**Pull request by kwaa**
## Description

Since Deno std provides toml parsing, it looks good to provide a TOML plugin.

### Check List

> Sorry, I'm not really good at writing tests.

- [x] Have you read the
      [CODE OF CONDUCT](https://github.com/lumeland/lume/blob/master/CODE_OF_CONDUCT.md)
- [x] Have you read the document
      [CONTRIBUTING](https://github.com/lumeland/lume/blob/master/CONTRIBUTING.md)
  - [x] One pull request per feature. If you want to do more than one thing,
        send multiple pull request.
  - [ ] Write tests.
  - [x] Run deno `fmt` to fix the code format before commit.
  - [x] Document any change in the `CHANGELOG.md`.

### CODE REVIEW

File: plugins/toml.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    data: string[];
+  };
+}
+
+// Default options
+export const defaults: Options = {
+  extensions: [".toml"],
+};
+
+/** A plugin to add support for YAML files */
```
Comment (oscarotero) [inline plugins/toml.ts:19]: Just a tiny thing: replace YAML with TOML in this comment

**Review by oscarotero — CHANGES_REQUESTED** [review]

**Comment by oscarotero** [comment]
Thanks!
Can you add a test for this feature?

**Comment by kwaa** [comment]
> Thanks! Can you add a test for this feature?

Unfortunately, I'm not very good at writing tests yet.

**Comment by oscarotero** [comment]
Okay, no problem, I will.

File: plugins/toml.ts (hunk 1 above)
Comment (kwaa) [reply plugins/toml.ts]: > Just a tiny thing: replace YAML with TOML in this comment

Thanks, didn't notice. Fixed!

**Pull request merged**